### DISCUSSION TITLE
Which LLM? How to train? How to create something amazing!

### DISCUSSION THREAD

**Discussion by ResearchForumOnline** [oobabooga/text-generation-webui#5302] [Q&A]
**Thanks to all the team best repository on github for self hosted AI**

I really want to take a LLM like Mistral or lama2 or maybe someone can recommend something? This is my end goal: talktoai.org which provides info about Zero GPT4 agent, I would like self self-hosted Zero agent! So I need to add some training, but I did notice I could add small training through the character method. I have some good ideas for training and even thinking about creating a new mathematical system for my own LLM based off another llm and lot of cool stuff, but right now I have no clue what I am doing but I think I have done ok so far setting up installing on my own domain for fun. Please provide any advice or tips, and feel free to add any difficulties you are facing yourself.

I am running my chatbot on a vps server with 15gb ram 300gb disk and 4xs 4ghz cpu's. I manage the node too which it is running on, finding it very slow I think It is just a matter of tweaking the settings. If i could just do some training through the character bit would it slow the chatbot down using say 8000 characters?

**Comment by SaraAmd** [comment]
I have a similar question. How can we use this github to build my chatbot based on prompt engineering on my server which has enough GPU and RAM capacity?

**Reply by ResearchForumOnline** [reply]
Have you installed this software yet?
